packages/react-catalog-view-extension/src/components/VerticalTabs/examples/VerticalTabs.md

```diff
@@ -61,9 +61,9 @@ class MockVerticalTabsExample extends React.Component {
         {renderTab('all', 'All', null, { shown: true })}
         {renderTab('one', 'Tab One', [
           renderTab('one-one', 'Tab One-One', [
-            renderTab('one-one-one', 'Tab One-One-One'),
-            renderTab('one-one-two', 'Tab One-One-Two'),
-            renderTab('one-one-three', 'Tab One-One-Three')
+            renderTab('one-one-one', 'About Modal', null, {href: "../components/about-modal"}),
+            renderTab('one-one-two', 'Accordion', null, {href: "../components/accordion"}),
+            renderTab('one-one-three', 'Redhat', null, {href: "http://www.redhat.com"})
```

Comment:
@tlabaj Is this what you mean by adding the prop to the demo app?

Reply:
I think she means add it under `packages/react-integration/demo-app-ts` and write a test under `packages/react-integration/cypress/integration` that clicks the link and makes sure the browser points to the specified `href`.